packages/client/tests/e2e/_utils/run.ts

```diff
@@ -198,7 +202,13 @@ async function restoreOriginalState() {
 }
 
 async function printFile(filePath: string) {
-  await pipeline(createReadStream(filePath), process.stdout)
+  try {
+    const fileStream = createReadStream(filePath)
+    fileStream.pipe(process.stdout, { end: false })
```

Comment:
ℹ️ By default piping closes the target stream once its done. In this case closing stdout. Which lead to no further log output happening. That caused a very confusing picture in the CI logs that send me in the wrong direction for a while...

Reply:
oh wow